include/core/boundary_conditions.h

```diff
@@ -745,6 +746,129 @@ namespace BoundaryConditions
     prm.leave_subsection();
   }
 
+  /**
+* @brief This class manages the boundary conditions for the Cahn-Hilliard solver
+* It introduces the boundary functions and declares the boundary conditions
+* coherently.
+*  - if bc type is "dirichlet" (Dirichlet condition), "value" is the
+* double passed to the deal.ii ConstantFunction
+
```

Comment:
It might be better to use only the FunctionParser instead of using a double.

Reply:
This will be taken into account in the next PR